lib/matplotlib/colors.py

```diff
@@ -796,9 +789,7 @@ def from_list(name, colors, N=256, gamma=1.0):
         return LinearSegmentedColormap(name, cdict, N, gamma)
 
     def _resample(self, lutsize):
-        """
-        Return a new color map with *lutsize* entries.
-        """
+        """Return a new color map with *lutsize* entries."""
```

Comment:
also inherited

Reply:
Given that it's a one-line docstring and the base function is not implemented, I'd just leave this in. The docstring is simple enough, and likely not to change so that the danger of getting out-of-sync is small. In that case, I think it's better to stay explicit and not redirect the docstring.